### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by mjdiloreto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+This repository contains an agent to play the game [_"The Binding of Issac: Afterbirth+"_](https://store.steampowered.com/agecheck/app/570660/).
+
+## Installation and Usage
+1. Install _"The Binding of Issac: Afterbirth+"_ through Steam.
+2. Clone this repository to `C:/Users/[your username]/My Games/Binding of Isaac Afterbirth+ Mods/`
+3. On Steam, set --luadebug in launch options by right clicking the game and going to Properties -> General -> Set Launch Options
+4. Run Game
+
+By default the agent will begin to play, you can disable this by pressing "R"
```
Comment (2-X) [inline README.md:12]: i switched it so that by default it doesn't play. Which do you prefer?

Comment (mjdiloreto) [reply README.md:12]: No preference to be honest. We can leave the read me as is and resolve it when we have a real decision.

Comment (2-X) [reply README.md:12]: deal

**Review by 2-X — APPROVED** [review]
LGTM

**Pull request merged**